ament_index_cpp/include/ament_index_cpp/get_resource.hpp

```diff
@@ -15,10 +15,10 @@
 #ifndef __ament_index_cpp__get_resource__h__
 #define __ament_index_cpp__get_resource__h__
 
-#include <string>
-
 #include <ament_index_cpp/visibility_control.h>
 
+#include <string>
+
```

Comment:
I do agree but could get it to pass that way. Can you update this PR accordingly while still passing ament_cpplint?